tests/smoketest/smoketest_collectd_entrypoint.sh

```diff
@@ -62,31 +62,28 @@ grep -E '"result":\[{"metric":{"__name__":"sensubility_container_health_status",
 metrics_result=$((metrics_result || $?))
 echo; echo
 
-if [ "$OBSERVABILITY_STRATEGY" != "use_redhat" ]; then
-  echo "*** [INFO] Get documents for this test from ElasticSearch..."
-  DOCUMENT_HITS=$(curl -sk -u "elastic:${ELASTICSEARCH_AUTH_PASS}" -X GET "${ELASTICSEARCH}/_search" -H 'Content-Type: application/json' -d'{
-    "query": {
-      "bool": {
-        "filter": [
-          { "term" : { "labels.instance" : { "value" : "'${CLOUDNAME}'", "boost" : 1.0 } } },
-          { "range" : { "generated" : { "gte" : "now-1m", "lt" : "now" } } }
-        ]
-      }
+echo "*** [INFO] Get documents for this test from ElasticSearch..."
```

Comment:
Final nit, Elasticsearch

Reply:
This one was pre-existing, even!

```suggestion
echo "*** [INFO] Get documents for this test from Elasticsearch..."
```